helix-view/src/editor.rs

```diff
@@ -238,6 +240,27 @@ impl std::str::FromStr for LineNumber {
     }
 }
 
+#[derive(Debug, Copy, Clone, PartialEq, Eq, Serialize, Deserialize)]
+#[serde(rename_all = "kebab-case")]
+pub enum GutterType {
+    /// Show diagnostics
```

Comment:
We should document that diagnostics also includes breakpoints both here and in the user doc